tst/SdpApiTest.cpp

```diff
@@ -962,8 +948,13 @@ a=rtpmap:102 H264/90000
 
 TEST_P(SdpApiTest_SdpMatch, populateSingleMediaSection_TestH264Fmtp)
 {
-    const std::string offer = sdpOffer();
-    PCHAR sdp = const_cast<PCHAR>(offer.c_str());
+    UINT64 sdpSize;
+    PCHAR pSdp;
+
+    ASSERT_EQ(STATUS_SUCCESS, readFile((PCHAR) filename(), TRUE, NULL, &sdpSize));
```

Comment:
It's fine but for more or less knowns sizes you could have used a static buffer instead of getting the size, alloc-ing and freeing. For the test it's more than fine.